java/src/org/openqa/selenium/ie/InternetExplorerDriver.java

```diff
@@ -130,22 +135,27 @@ public InternetExplorerDriver(InternetExplorerDriverService service) {
    * @param options The options required from InternetExplorerDriver.
    */
   public InternetExplorerDriver(InternetExplorerDriverService service,
-                                InternetExplorerOptions options) {
+                                InternetExplorerOptions options,
+                                ClientConfig clientConfig) {
     if (options == null) {
       options = new InternetExplorerOptions();
     }
     if (service == null) {
       service = InternetExplorerDriverService.createDefaultService();
     }
-    run(service, options);
+    if (clientConfig == null){
+      clientConfig = ClientConfig.defaultConfig();
+    }
+
+    run(service, options, clientConfig);
   }
 
   @Beta
   public static RemoteWebDriverBuilder builder() {
     return RemoteWebDriver.builder().oneOf(new InternetExplorerOptions());
   }
 
-  private void run(InternetExplorerDriverService service, Capabilities capabilities) {
+  private void run(InternetExplorerDriverService service, Capabilities capabilities, ClientConfig clientConfig) {
     assertOnWindows();
 
     setCommandExecutor(new DriverCommandExecutor(service));
```

Comment:
Even though we are getting a `ClientConfig` object, we are not using it. Please help add an overloaded constructor into `DriverCommandExecutor` that takes in a DriverService and a ClientConfig object and then internally invokes the other relevant constructors and here we should be calling that new constructor wherein we pass the client Config object.